### PR TITLE
updated login activity user interface

### PR DESCRIPTION
**Pull request by dakshverma2411**
updated the user interface the way you asked it
Here's a screenshot
Screenshot 2020-10-26 142241

### CODE REVIEW

**Comment by dakshverma2411** [comment]
I apologize that earlier I didn't see what was your requirement.

File: app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -4,45 +4,120 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/white"
     tools:context=".LoginActivity">
 
-    <EditText
-        android:id="@+id/editTextTextPersonName"
+    <ImageView
```
Comment (starboi02) [inline app/src/main/res/layout/activity_login.xml]: Do not use image use Text instead

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -4,45 +4,120 @@
...
+        android:layout_height="80dp"
+        android:layout_marginTop="70dp"
+        android:src="@drawable/login_title"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="MissingConstraints" />
+
+    <ImageView
+        android:id="@+id/imageView"
```
Comment (starboi02) [inline app/src/main/res/layout/activity_login.xml:28]: width and height should be same as that of figma link

File: app/src/main/res/layout/activity_login.xml [hunk 3]
```diff
@@ -4,45 +4,120 @@
...
+        app:layout_constraintVertical_bias="0.026"
+        tools:ignore="MissingConstraints" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Email"
+        android:fontFamily="@font/reemkufi_regular"
+        android:textColor="@android:color/darker_gray"
+        android:textSize="12sp"
```
Comment (starboi02) [inline app/src/main/res/layout/activity_login.xml]: text size should be 14sp

File: app/src/main/res/layout/activity_login.xml [hunk 4]
```diff
@@ -4,45 +4,120 @@
...
+    <EditText
+        android:id="@+id/editTextTextPersonName"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_marginLeft="50dp"
+        android:layout_marginRight="50dp"
+        android:backgroundTint="#2F80B8"
+
         android:ems="10"
+        android:fontFamily="@font/reemkufi_regular"
```
Comment (starboi02) [inline app/src/main/res/layout/activity_login.xml]: wrong font

File: app/src/main/res/layout/activity_login.xml [hunk 5]
```diff
@@ -4,45 +4,120 @@
...
+        app:layout_constraintBottom_toTopOf="@+id/editTextTextPassword"
+        app:layout_constraintStart_toStartOf="@+id/editTextTextPassword"
+        tools:ignore="MissingConstraints" />
+
+
+    <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:text="Forgot Password ?"
+        android:fontFamily="@font/reemkufi_regular"
```
Comment (starboi02) [inline app/src/main/res/layout/activity_login.xml]: wrong font

**Review by starboi02 — CHANGES_REQUESTED** [review]
@dakshverma2411 please try to make it as similar as possible to that of the figma link. Use fonts, text size and margins defined in the figma link.

**Comment by dakshverma2411** [comment]
fixed them.

**Review by starboi02 — APPROVED** [review]

**Pull request merged**